db/migrate/20200609153929_drop_emergency_login_keys_if_exists.rb

```diff
@@ -0,0 +1,22 @@
+class DropEmergencyLoginKeysIfExists < ActiveRecord::Migration[5.2]
```

Comment:
This migration should not exist

Reply:
This was to prevent a failed deployment - see the [commit message](https://github.com/DFE-Digital/teacher-vacancy-service/pull/1647/commits/342ee387ed40a172b516c255344edf328738bc2f). How can I better guard against this error appearing on production?

Reply:
There is not going to be this problem on production. I would suggest to remove this migration, wait for the syncing of databases tonight where we remove the data and recreate the tables, and deploy it again first thing tomorrow in the morning.

Reply:
I removed the migration, and yes, no issues.